packages/office-addin-project/src/convert.ts

```diff
@@ -91,7 +91,11 @@ function updatePackages(): void {
 
   for (let i = 0; i < depedentPackages.length; i++) {
     const depedentPackage = depedentPackages[i];
-    command += ` ${depedentPackage}@latest`;
+    if (depedentPackage === "@microsoft/teamsfx-cli") {
```

Comment:
Currently, teamsfx should be a nested dependency and not a direct dependency so this shouldn't be necessary and the teamsfx entry in the array should be removed.

Reply:
We still need the dependency on @microsoft/teamsfx-cli right? I just updated to require it

Reply:
The project has a dependency on office-addin-debugging which depends on office-addin-dev-settings which depends of teamsfx-cli.  I don't think we need to require teamsfx-cli at the project level package.json directly.

Reply:
You should specify the dependencies based on the direct package that the code uses. If it DIRECTLY gets code from teamsfx-cli, even if another dependent package, always does, you should still have a dependency on it.